app/src/main/res/values/strings.xml

```diff
@@ -460,7 +460,7 @@
         <item>Medium</item>
         <item>Large</item>
     </string-array>
-    <string name="style_choose_title">App theme</string>
+    <string name="style_choose_title">Theme</string>
```

Comment:
```suggestion
    <string name="style_choose_title">App theme</string>
```
Still think the theme is the theme and that database theme is "Database theme", but alas.